src/katello/enabled_report.py

```diff
@@ -0,0 +1,33 @@
+class EnabledReport(object):
```

Comment:
Looks like we can remove all of the other EnabledReport classes and use this one now

Reply:
Looking into that next.